app/src/main/java/com/sapuseven/untis/activities/SettingsActivity.kt

```diff
@@ -255,6 +257,25 @@ class SettingsActivity : BaseComposeActivity() {
 												dataStore = dataStorePreferences.flingEnable
 											)
 										}
+										if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+											PreferenceCategory(stringResource(id = R.string.preference_app_language)) {
```

Comment:
Please use a separate string resource for the category.
1. In English, categories are capitalized while settings are lowercase ("App Language" vs "App language")
2. We might want to add other interface settings in the future, so they should be independent